sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -47,364 +48,444 @@ const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
-            } catch (err) {
-              response = err;
-            }
-            break;
-
-          case EntityType.TOPIC:
-            try {
-              response = await createEntity(entityType, alwaysBeExistingTopic);
-            } catch (err) {
-              response = err;
-            }
-
-            break;
-
-          case EntityType.SUBSCRIPTION:
-            try {
-              response = await createEntity(
-                entityType,
-                alwaysBeExistingSubscription,
-                alwaysBeExistingTopic
-              );
-            } catch (err) {
-              response = err;
-            }
-
-            break;
-
-          case EntityType.RULE:
-            try {
-              response = await createEntity(
-                entityType,
-                alwaysBeExistingRule,
-                alwaysBeExistingTopic,
-                alwaysBeExistingSubscription
-              );
-            } catch (err) {
-              response = err;
-            }
-
-            break;
-
-          default:
-            throw new Error("TestError: Unrecognized EntityType");
-        }
+[
+  {
+    entityType: EntityType.QUEUE,
+    alwaysBeExistingEntity: alwaysBeExistingQueue,
+    alwaysBeDeletedEntity: alwaysBeDeletedQueue,
+    parentTopicName: undefined,
+    parentSubscriptionName: undefined
+  },
+  {
+    entityType: EntityType.TOPIC,
+    alwaysBeExistingEntity: alwaysBeExistingTopic,
+    alwaysBeDeletedEntity: alwaysBeDeletedTopic
+  },
+  {
+    entityType: EntityType.SUBSCRIPTION,
+    alwaysBeExistingEntity: alwaysBeExistingSubscription,
+    alwaysBeDeletedEntity: alwaysBeDeletedSubscription,
+    parentTopicName: alwaysBeExistingTopic
+  },
+  {
+    entityType: EntityType.RULE,
+    alwaysBeExistingEntity: alwaysBeExistingRule,
+    alwaysBeDeletedEntity: alwaysBeDeletedRule,
+    parentTopicName: alwaysBeExistingTopic,
+    parentSubscriptionName: alwaysBeExistingSubscription
+  }
+].forEach((testCase) => {
+  describe(`Atom management - Basic CRUD on "${testCase.entityType}" entities #RunInBrowser`, function(): void {
+    it(`Lists on non-existing entities for type ${testCase.entityType} returns empty array successfully`, async () => {
+      let response;
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          testCase.alwaysBeExistingEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        log.httpAtomXml("Ignoring clean up step");
+      }
 
-        should.equal(response.statusCode, 409, "Error must not be undefined");
-      });
+      response = await listEntities(
+        testCase.entityType,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
 
-      it(`Lists available ${entityType} entities successfully`, async () => {
-        let response;
+      try {
+        await createEntity(
```

Comment:
I'm not sure why you're creating an entity here since the test is checking that `listEntities` returns a result if no entities exist, and nothing is asserted on `createEntity` anyway.

Reply:
Idea was that "alwaysBeExisting" entities are always existing for tests that rely on their existence. 

These tests will be removed though as discussed in yesterdays meeting.